node/clients/biggy-search.ts

```diff
@@ -269,6 +270,7 @@ export class BiggySearchClient extends ExternalClient {
       fuzzy,
       locale: this.locale,
       bgy_leap: leap ? true : undefined,
+      allowRedirect:  options?.allowRedirect == undefined ? true : options?.allowRedirect ,
```

Comment:
```suggestion
      allowRedirect:  options?.allowRedirect === false ? false : true,
```
I think it does the same thing but it might be a little more readable. what do you think?